modules/integration/tests-integration/tests-service/src/test/resources/artifacts/ESB/synapseconfig/esbjava2283/api.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<api xmlns="http://ws.apache.org/ns/synapse"
+     name="serviceTest"
+     context="/serviceTest">
+    <resource methods="GET" uri-template="/test*">
+        <inSequence>
+            <log level="full"/>
```

Comment:
Log full was added just to display request content in the console and removed that since it not required.